src/editor.js

```diff
@@ -1716,13 +1725,64 @@ export class OdooEditor extends EventTarget {
         }
     }
 
+    /**
+     * Convert valid url text into links inside the given element.
+     *
+     * @param {HTMLElement} el
+     */
+    _checkForUrlInElement(el) {
```

Comment:
It's not a check, it is as converter as the docstring suggest. `_convertUrlInElement` ?